changelog.md

```diff
@@ -35,6 +35,7 @@
 * [3155](https://github.com/zeta-chain/node/pull/3155) - fix potential panic in the Bitcoin inscription parsing
 * [3162](https://github.com/zeta-chain/node/pull/3162) - skip depositor fee calculation if transaction does not involve TSS address
 * [3179](https://github.com/zeta-chain/node/pull/3179) - support inbound trackers for v2 cctx
+* [3184](https://github.com/zeta-chain/node/pull/3184) - zetaclient should not retry if inbound vote message validation fails
```

Comment:
To be moved above under `## Unreleased`

Reply:
moved: https://github.com/zeta-chain/node/pull/3184/commits/33fd7d37680414d862fefece193acdae98f62817